src/Application.php

```diff
@@ -58,6 +58,14 @@ public function __construct(ContainerBuilder $containerBuilder = null, array $va
                 ])
             );
         });
+
+        // Override the callback resolver with ours
+        $this['callback_resolver'] = $this->share(function () {
+            return new CallbackResolver(
+                $this,
+                new CallableResolver($this->containerInteropProxy)
```

Comment:
Could the callable resolver from line 54 be reused here? I don't think there's a need to create it twice.